README.md

```diff
@@ -1,262 +1,289 @@
+	<a href="https://github.com/runem/web-component-analyzer/graphs/contributors"><img alt="Contributors" src="https://img.shields.io/github/contributors/runem/web-component-analyzer.svg" height="20"/></a>
+</p>
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/5372940/68087781-1044ce80-fe59-11e9-969c-4234f9287f1b.gif" alt="Web component analyzer GIF"/>
+</p>
+
+`web-component-analyzer` is a CLI that makes it possible to easily analyze web components. It analyzes your code and jsdoc in order to extract `properties`, `attributes`, `methods`, `events`, `slots`, `css shadow parts` and `css custom properties`. Works with both javascript and typescript.
+
+Try the online playground [here](https://runem.github.io/web-component-analyzer/)
+
+In addition to [vanilla web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) this tool supports web components built with the following libraries:
+
+- [lit-element](https://github.com/Polymer/lit-element)
+- [polymer](https://github.com/Polymer/polymer)
+- [stencil](https://github.com/ionic-team/stencil) (partial)
+- [lwc](https://github.com/salesforce/lwc)
+- [open an issue for library requests](https://github.com/runem/web-component-analyzer/issues)
+
+[![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/colored.png)](#installation)
+
+## ➤ Installation
+
+<!-- prettier-ignore -->
+```bash
+$ npm install -g web-component-analyzer
+```
+
+**or**
+
+<!-- prettier-ignore -->
+```bash
+$ npx web-component-analyzer src
+```
+
+[![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/colored.png)](#usage)
+
+## ➤ Usage
+
+<!-- prettier-ignore -->
+```bash
+$ wca analyze
+$ wca analyze src --format markdown
+$ wca analyze "src/**/*.{js,ts}" --outDir components
+$ wca analyze my-element.js --outFile custom-elements.json
+$ wca analyze --outFiles {dir}/custom-element.json
+```
+
+<img src="https://user-images.githubusercontent.com/5372940/54445420-02fd9700-4745-11e9-9305-47d6ec3c6307.gif" />
+
+The analyze command analyses an optional `<input glob>` and emits the output to the console as default. When the `<input glob>` is omitted it will find all components excluding `node_modules`. The default format is `markdown`.
+
+### Options
+
+<!-- prettier-ignore -->
+| Option                      | Type                     | Description                                                                  |
+|-----------------------------|--------------------------| ---------------------------------------------------------------------------- |
+| `--format <format>`         | `markdown` \             | `json` \| `vscode` \| `webtypes` | Specify output format. Default is `markdown`.                                |
```

Comment:
i think this got a bit mangled at some point. the `\` isn't escaping the vertical bar anymore since there's newly added whitespace after it. so the type is in the wrong column

Reply:
I think you are right, I broke README. Full file end lines got changed changed by pre-commit hook, and also IDE rebuilt table formatting without understanding `|` column escaping, which has not been seen due to full file line ending change.

Can you please commit line ending change or disable line ending change on pre-commit hook on README on master branch ? Currently whatever the change I make, full README comes in change.
I will then remove any change on README not related to web-types.

Reply:
for now i would check what the line endings are in the existing file and keep them that way, then fix up what your editor did to the columns here.

even if you reformatted it to LF (assuming thats what happened here, CRLF -> LF), it'd be fine as long as you don't mangle the table and what not.

this PR is gonna take some time to land as the only other maintainers are pretty busy most of the time. so depending on another PR to change formatting etc isn't gonna be ideal i think (though we may still need to do that separately).

Reply:
I was able to avoid to avoid commit hook changing line endings by removing package.json and node_modules before commiting. Readme is fixed, with a clean diff with master branch